key_value_store/key_value_service.cc

```diff
@@ -58,18 +79,85 @@ Status KeyValueStoreImpl::get(ServerContext* context,
 Status KeyValueStoreImpl::remove(ServerContext* context, 
                                      const RemoveRequest* request, RemoveReply* reply) {
   if (storage_.DeleteKey(request->key())) {
+    // If persistent mode is activated, append remove request into file.
+    if (persistent_mode_) {
+      Persistent_store("remove", request->key(), "");
+    }
     return Status::OK;
   }
   return Status(StatusCode::NOT_FOUND, "Failed to delete the key.");
 }
 
+// When instantiating a KeyValueStore class, load data from persistent_db_
+// if the file exists.
+void KeyValueStoreImpl::Persistent_load() {
+  if (persistent_db_.length() == 0) {
+    persistent_mode_ = false;
+    return;
+  }
+  
+  persistent_mode_ = true;
+  std::ifstream in_file;
+  in_file.open(persistent_db_);
+  // If file doesn't exist before, create it.
+  if (!in_file.is_open()) {
+    std::cout << "Create new file " << persistent_db_ << " for persistent store..." << std::endl;
+    std::ofstream out_file;
+    out_file.open(persistent_db_);
+    out_file.close();
+    std::cout << "Successfully created." << std::endl;
+    return;
+  }
+  /* 
+    Read data from persistent_db_. Key-Value pairs in `put` request are separated into four
+    parts. 1. # of chars for key; 2. Key; 3. # of chars for value; 4. Value;
+    For `remove` request, Only key is stored. Only first two parts are stored.
+  */
```

Comment:
Keep comments consistent with the // syntax

Reply:
Fixed.